doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -0,0 +1,545 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "MNwt9bSG0hin"
+   },
+   "source": [
+    "# Performing GPU Batch Prediction on Images with a PyTorch Model\n",
+    "\n",
+    "In this example, we will introduce how to use the Ray AIR {class}`BatchPredictor <ray.train.batch_predictor.BatchPredictor>` for **large-scale batch inference with multiple GPU workers.**\n",
+    "\n",
+    "For this demo, we will use the [Imagenette](https://github.com/fastai/imagenette) dataset, which is a subset of Imagenet with 10 classes.\n",
```

Comment:
In this notebook, we will:

1. Load Imagenette dataset from S3 bucket and create a ray dataset
2. Load a pretrained ResNet model and build a checkpoint
3. Define a preprocessor
4. Construct a BatchPredictor with the checkpoint and preprocessor
5. Do batch prediction on multiple GPUs.
6. Evaluate the predictions and save results to S3/local disk.

Reply:
Updated to the notebook！